# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLFramebuffer.h
# Commit: 6e9a89ead3a5
# Full Hash: 6e9a89ead3a52864c48a4e5481e9a35e4aad5e98
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r dd00e2da3a0f -r 6e9a89ead3a5 dom/canvas/WebGLFramebuffer.h
--- a/dom/canvas/WebGLFramebuffer.h	Wed Jun 10 22:21:02 2020 +0000
+++ b/dom/canvas/WebGLFramebuffer.h	Wed Jun 10 22:21:04 2020 +0000
@@ -10,6 +10,7 @@
 
 #include "mozilla/WeakPtr.h"
 
+#include "GLScreenBuffer.h"
 #include "WebGLObjectModel.h"
 #include "WebGLStrongTypes.h"
 #include "WebGLTexture.h"
@@ -138,6 +139,7 @@
   const GLuint mGLName;
   bool mHasBeenBound = false;
   const UniquePtr<gl::MozFramebuffer> mOpaque;
+  gl::SwapChain mOpaqueSwapChain;
   bool mInOpaqueRAF = false;
 
  private: